CONTRIBUTING.md

```diff
@@ -202,6 +230,13 @@ Chunks preferably shouldn't be named.
 If we do end up using [bookdown](https://bookdown.org/yihui/bookdown/) at some point, repetitive chunk names like `import data` will cause havoc.
 Plus its just another thing to have to keep track of.
 
+#### Code Style
+
+These analyses follow the [Google R Style Guide](http://web.stanford.edu/class/cs109l/unrestricted/resources/google-style.html) which is based on the tidyverse style guide.
```

Comment:
I believe `styler` is enforcing the tidyverse guide, so we should probably use the tidyverse guide and link that here? Or, at this point, these guides may be sufficiently similar that it doesn't matter.

Reply:
> I believe `styler` is enforcing the tidyverse guide, so we should probably use the tidyverse guide and link that here? Or, at this point, these guides may be sufficiently similar that it doesn't matter.

This google link starts off with linking to tidyverse anyway so it's almost like both are linked. I just like the Google layout better. But they do have so much overlap by now, it doesn't matter much.

Reply:
That link is the outdated guide. I think you want this one: https://google.github.io/styleguide/Rguide.html 

Though that guide includes:

> Google prefers identifying functions with `BigCamelCase` to clearly distinguish them from other objects.

Is that the plan? (I think custom functions are unlikely to come up often, to be fair.)

Reply:
They also disagree on `return()` statements, though here I am a bit more on Google's side. I like explicit returns. https://style.tidyverse.org/functions.html#return

Reply:
Odd that the google link isn't a full style guide anymore.

Reply:
> They also disagree on `return()` statements, though here I am a bit more on Google's side. I like explicit returns. 

Agree with this!

Reply:
I don't know if custom functions will come up (maybe in Advanced Topics) but I'm fine with following Google still. (But I'll update the link out here).